### PR TITLE
Add details about solver rewards and payouts

### LINKED ISSUE #137: chore: Improve Solver auction and rewards
### Background

This page is missing important content

### Acceptance criteria

- [x] Add link to solver rewards Dune dashboard and the repository that pays out the rewards (as well as information about the payout cycle)
- [x] Add concrete information about the current CoW amounts as well as performance/consistency rewards split
- [x] Add section explaining rewards for quotes

### PR DESCRIPTION
**Pull request by harisang**
This PR extends the page about the solver rewards mechanism and the payouts.

Fixes #137

### CODE REVIEW

**Comment by harisang** [comment]
@mfw78 sorry i believe i accidentally tagged you for a review. feel free to ignore this.

File: docs/cow-protocol/reference/core/auctions/rewards.md
Comment (mfw78) [inline docs/cow-protocol/reference/core/auctions/rewards.md]:
```suggestion
:::note

For the interested reader, the main source of truth for the weekly payments to solvers is this [Dune dashboard](https://dune.com/cowprotocol/cow-solver-rewards?StartTime=2023-11-28+00%3A00%3A00&EndTime=2023-12-05+00%3A00%3A00&EndTime_d3bba3=2023-12-05+00%3A00%3A00&EndTime_dd004a=2023-12-05+00%3A00%3A00&StartTime_d3f009=2023-11-28+00%3A00%3A00).

:::
```

File: docs/cow-protocol/reference/core/auctions/rewards.md
Comment (mfw78) [inline docs/cow-protocol/reference/core/auctions/rewards.md]:
```suggestion
The auction's goal is to reward solvers based on how much value they provide to the users. In each auction cycle, solvers compete by proposing a solution to settle the batch, and the winning solver receives a reward that is part in `ETH` and part in `COW` (see below).

:::note

The reward in ETH is not part of the rewards budget, and is meant to cover the costs associated with the gas spent by each solver; importantly, this ETH is obtained by converting the gas fees user orders pay and so have nothing to do with the COW rewards budget.

:::
```

File: docs/cow-protocol/reference/core/auctions/rewards.md
Comment (mfw78) [inline docs/cow-protocol/reference/core/auctions/rewards.md:26]:
```suggestion
```

File: docs/cow-protocol/reference/core/auctions/rewards.md
Comment (mfw78) [inline docs/cow-protocol/reference/core/auctions/rewards.md]:
```suggestion
As specified in [CIP-27](https://snapshot.org/#/cow.eth/proposal/0x64e061568e86e8d2eec344d4a892e4126172b992cabe59a0b24c51c4c7e6cc33), solvers that participate in the price estimation competition get rewarded as follows. Each market order is associated with a quote that is responsible for determining the gas fee attached to the order as well as the final limit price of the order. The protocol keeps track of the quote associated with each created order and the corresponding solver that provided the quote, and if and when the order gets executed, the solver that provided the quote (that can be different compared to the solver that ended up executing the order) gets rewarded with a fixed 9 COW reward.
```

**Review by mfw78 — COMMENTED** [review]
Minor editorial review, typos and admonitions

File: docs/cow-protocol/reference/core/auctions/rewards.md
Comment (harisang) [reply docs/cow-protocol/reference/core/auctions/rewards.md]: Ooops, missed those. Sooorry!

**Comment by harisang** [comment]
> Minor editorial review, typos and admonitions

Thanks a lot. All very useful, and applied the changes already.

**Comment by mfw78** [comment]
Will wait a day or so for @fhenneke to review if there is a subject matter issue to be edited. Perhaps @fleupold, as the original issue creator may also have something to add?

**Comment by harisang** [comment]
> Will wait a day or so for @fhenneke to review if there is a subject matter issue to be edited. Perhaps @fleupold, as the original issue creator may also have something to add?

Yeah i also want to make another pass, and would definitely wait for @fhenneke to review before thinking about merging.

**Review by fhenneke — APPROVED** [review]
The changes look good to me.

**Pull request merged**
